packages/core/react/use-assistant.ts

```diff
@@ -83,8 +91,10 @@ export function experimental_useAssistant({
 
     const result = await fetch(api, {
       method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
+      credentials,
+      headers: { ...headers, 'Content-Type': 'application/json' },
```

Comment:
Can you change the order (`...headers` later so it can override the content type)?